templates/patterns/nav/pattern-nav.html.twig

```diff
@@ -8,4 +8,5 @@
   'tabs': tabs,
   'tabs_content': tabs_content,
   'navbar': navbar,
-} %}
+  'attributes': attributes
```

Comment:
removing this and adding this doesn't have any effect visually as far as I can see, what is it for ?

Reply:
I'm doing this now in all patterns I'm reviewing when I add "only" to the pattern definition. I had to do this to allow attributes to be passed to the component.